--- stackwalker_win.cpp
+++ stackwalker_win.cpp	2008-06-03 17:37:38.000000000 +0200
@@ -353,7 +353,7 @@
     }
     _ftprintf(stderr, _T("(%s Symbols Loaded)")                   , szSymbolType);
     _ftprintf(stderr, _T("\n"));
-    _ftprintf(stderr, _T("    Linked PDB Filename   : %s\n")      , Module.CVData);
+//     _ftprintf(stderr, _T("    Linked PDB Filename   : %s\n")      , Module.CVData);
     if (bFileVersionSupported && bFileVersionRetrieved) {
         _ftprintf(stderr, _T("    File Version          : %s\n")  , szFileVersion);
         _ftprintf(stderr, _T("    Company Name          : %s\n")  , szCompanyName);
@@ -657,7 +657,7 @@
 // #################################################################################
 // Here the Stackwalk-Part begins.
 //   Some of the code is from an example from a book 
-//   But I couldn´t find the reference anymore... sorry...
+//   But I couldnï¿½t find the reference anymore... sorry...
 //   If someone knowns, please let me know...
 // #################################################################################
 // #################################################################################
