--- Makefile.mingw
+++ Makefile.mingw.new	2008-06-11 15:08:56.000000000 +0200
@@ -1,6 +1,6 @@
-BOINCDIR = ./boinc
+BOINCDIR = $(BOINC_SRC)
 DEBUG = -D__DEBUG__ -g3
-OBJ  = boinc_api.o util.o win_util.o app_ipc.o diagnostics.o diagnostics_win.o filesys.o hostinfo.o md5.o md5_file.o mem_usage.o mfile.o miofile.o parse.o prefs.o proxy_info.o str_util.o shmem.o stackwalker_win.o base64.o
+OBJ  = graphics2.o graphics2_util.o boinc_api.o util.o win_util.o app_ipc.o diagnostics.o diagnostics_win.o filesys.o hostinfo.o md5.o md5_file.o mem_usage.o mfile.o miofile.o parse.o prefs.o proxy_info.o str_util.o shmem.o stackwalker_win.o base64.o
 LINKOBJ  = $(OBJ)
 LDFLAGS = -lwinmm -march=i386
 INCS = -I"$(BOINCDIR)" -I"$(BOINCDIR)/db" -I"$(BOINCDIR)" -I"$(BOINCDIR)/lib" -I"$(BOINCDIR)/api" 
