--- OGLFT.h.cmake
+++ OGLFT.h.cmake.new	2008-05-23 13:20:10.000000000 +0200
@@ -42,12 +42,21 @@
 #include <windows.h>
 #endif
 
+#ifdef __APPLE__
+#include <OpenGL/gl.h>
+#include <OpenGL/glu.h>
+#else
 #include <GL/gl.h>
 #include <GL/glu.h>
+#endif
 
 #ifndef OGLFT_NO_SOLID
+#ifdef __APPLE__
+#include <OpenGL/gle.h>
+#else
 #include <GL/gle.h>
 #endif
+#endif
 
 #ifndef OGLFT_NO_QT
 #define OGLFT_QT_VERSION @DESIRED_QT_VERSION@
@@ -97,7 +106,11 @@
   };
 
   //! Callback from GLU tessellation routines.
+#ifdef __APPLE__
+  typedef GLvoid (*GLUTessCallback)(...);
+#else
   typedef void (*GLUTessCallback)();
+#endif
 
   //! The FreeType library instance.
   /*!
